src/react/ISV/components/__tests__/ISVConfiguration.test.tsx

```diff
@@ -0,0 +1,446 @@
+
+      // Select existing IAM user
+      await userEvent.click(screen.getByText('Use an existing IAM User'));
+
+      expect(
+        screen.getByText('Generate a new set of AK/SK'),
+      ).toBeInTheDocument();
+    });
+  });
+
+  describe('Form Submission', () => {
+    it('should call setConfig with correct data on submit', async () => {
+      renderComponent(Commvault);
+
+      // Fill in account name
+      await userEvent.click(screen.getByText('Create a new account'));
+      const accountNameInput = screen.getByRole('textbox', {
+        name: /Account Name/i,
+      });
+      await userEvent.clear(accountNameInput);
+      await userEvent.type(accountNameInput, 'new-test-account');
+
+      // Fill in bucket name
+      const bucketNameInput = screen.getByRole('textbox', {
+        name: /Bucket name * /i,
+      });
+      await userEvent.clear(bucketNameInput);
+      await userEvent.type(bucketNameInput, 'test-bucket-1');
+
+      // Wait for form validation to complete
+      const submitButton = screen.getByRole('button', {
+        name: /Continue/i,
+      });
+
+      // Make sure the button is enabled
+      await waitFor(() => {
+        expect(submitButton).not.toBeDisabled();
+      });
+
+      // Submit the form
+      await userEvent.click(submitButton);
+
+      // Verify setConfig was called with correct data
+      await waitFor(() => {
+        expect(mockSetConfig).toHaveBeenCalledWith({
+          accountName: 'new-test-account',
+          accountNameType: 'create',
+          buckets: [
+            {
+              name: 'test-bucket-1',
+              tag: 'commvault',
+              capacity: '0',
+              capacityUnit: '1099511627776',
+            },
+          ],
+          enableImmutableBackup: true,
+        });
+      });
+    });
+  });
+
+  describe('Navigation', () => {
+    beforeEach(() => {
+      mockNavigate.mockReset();
+    });
+
+    it('should show skip modal when clicking Skip button', async () => {
+      renderComponent();
+
+      // Click the skip button
+      await userEvent.click(screen.getByText('Skip Use case configuration'));
+
+      // Verify modal is shown
+      expect(screen.getByText('Exit Veeam assistant?')).toBeInTheDocument();
+    });
+
+    // it('should navigate to accounts page when confirming skip', async () => {
+    //   renderComponent();
+
+    //   // Click the skip button
+    //   await userEvent.click(screen.getByText('Skip Use case configuration'));
+
+    //   // Click the "Exit" button in the modal
+    //   await userEvent.click(
+    //     screen.getByRole('button', { name: 'Exit configuration' }),
+    //   );
+
+    //   await waitFor(() => {
+    //     expect(mockNavigate).toHaveBeenCalledWith('/accounts');
+    //   });
+    // });
+
+    it('should close modal without navigating when canceling skip', async () => {
+      // Reset mocks before test
+      mockNavigate.mockClear();
```

Comment:
Should be done in a befroeEach, tests have to remains independant from each others